draft-kazuho-httpbis-priority.md

```diff
@@ -201,8 +201,9 @@ used.
 The background urgency (value 6) is used for responses of which the delivery can
 be postponed without having an impact on using other responses.
 
-As an example, a software update running in the background would be assigned the
-background urgency.
+As an example, the download of a large file in a web browser would be assigned
+the background urgency so it would not impact further page loads on the same
+domain.
```

Comment:
Maybe there is a more correct phrase than "same domain"?

Reply:
I agree. Options:
- same connection
- same host
- to the same origin

Reply:
I think "connection" would be fine.

The concern here might be that "connection" implies hop-by-hop rather than end-to-end, but I would not be too worried, considering that this is an example. If we want to describe how the priority header field can be used in hops other than the one that originates from the end client, I think there'd be a place better than here.

Reply:
same connection is most appropriate I think.